spec/policies/review_policy_spec.rb

```diff
@@ -0,0 +1,35 @@
+require_relative "../../app/policies/review_policy"
+
+RSpec.describe ReviewPolicy do
+  describe "#authorize_for_review?" do
+    it "returns true if user hasn't written a review yet" do
+      reviewer = double("user", present?: true)
+      reviews = []
+
+      valid_authorization = ReviewPolicy.new(reviews, reviewer)
+        .authorized_for_review?
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.